torch/csrc/distributed/rpc/rpc_agent.h

```diff
@@ -93,6 +119,18 @@ class TORCH_API RpcAgent {
       const WorkerInfo& to,
       Message&& message) = 0;
 
+  // At-least once semantics for send. Retries sending the message until an ACK
```

Comment:
Does ACK in this case mean that the future has been successfully processed and marked as completed? If so we might want to clarify, since ACK can also frequently means that the receval has been acknowledged but the worke has not actually been done